recipes/nvidia-apex/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "nvidia-apex" %}
+{% set version = "0.1" %}
+{% set commit = "088985936518be7e25795a30d8ab33affa9db6ed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/apex/archive/{{ commit }}.tar.gz
+  sha256: cc5ce27aa30b5ade54d520fafc1cd7d293dfba58c51851759a034d03e3f370bb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+  skip: True  # [osx or (not py36) or (win and vc<14)]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pytorch
+    - setuptools
+    - pip
+  run:
+    - python
+    - pytorch
+    - cxxfilt >=0.2.0  # [linux]
+    - tqdm >=4.28.1
+    - numpy >=1.15.3
+    - PyYAML >=5.1
+    - pytest >=3.5.1
```

Comment:
These pins need to be as loose as possible. I am skeptical of the tqdm pin in particular. Can you check them?